app/views/steps/applicant/address_details/edit.html.erb

```diff
@@ -1,24 +1,25 @@
 <% title t('.page_title') %>
+<% step_header %>
 
-<div class="grid-row">
-  <div class="column-two-thirds">
-    <%= step_header %>
+<div class="govuk-grid-row">
+  <div class="govuk-grid-column-two-thirds">
+    <%= govuk_error_summary %>
 
-    <h1 class="heading-xlarge gv-u-heading-xxlarge"><%=t '.heading', name: @form_object.record.full_name %></h1>
+    <h1 class="govuk-heading-xl"><%= t('.heading', name: @form_object.record.full_name) %></h1>
 
-    <%= step_form @form_object do |f| %>
-      <%= f.text_field :address_line_1 %>
-      <%= f.text_field :address_line_2, label_options: { class: 'visually-hidden' } %>
-      <%= f.text_field :town %>
-      <%= f.text_field :country %>
-      <%= f.text_field :postcode %>
+    <%= step_form @form_object, builder: GOVUKDesignSystemFormBuilder::FormBuilder do |f| %>
+      <%= f.govuk_text_field :address_line_1 %>
+      <%= f.govuk_text_field :address_line_2, label: { hidden: true } %>
+      <%= f.govuk_text_field :town %>
```

Comment:
Forgot to mention before, but we will need to set the witdth of town, country and postcode, as per example here:
https://design-system.service.gov.uk/patterns/addresses/multiple/index.html

I reckon for postcodes we've been using `width: 'one-third'` so for country and town we can use `width: 'one-half'` maybe? Give it a try and see how it feels.